.gitattributes

```diff
@@ -0,0 +1,5 @@
+# SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+* text=auto
```

Comment:
please do not blindly set `* text=auto`. You can refer to https://stackoverflow.com/questions/21472971/what-is-the-purpose-of-text-auto-in-gitattributes-file

What's the reason you needed this in the first place?